mm2src/mm2_bitcoin/chain/src/block_header.rs

```diff
@@ -1034,6 +1040,24 @@ mod tests {
         assert_eq!(serialized.take(), headers_bytes);
     }
 
+    #[test]
+    fn test_nmc_block_headers_serde_11() {
+        // NMC block headers
+        // start - #622807
+        // end - #622796
+        // Ref: https://chainz.cryptoid.info/nmc/block.dws?622807.htm
```

Comment:
This is a very nice block explorer that shows the raw header with it's fields and the same for transactions for a lot of coins, this should be handy in a lot of similar serialization/deserialization issues :)

Reply:
Right, it helped a lot while debugging 💯💯